Assets/HoloToolkit/Input/Scripts/Utilities/Interactions/TriggerButton.cs

```diff
@@ -3,7 +3,6 @@
 
 using System;
 using UnityEngine;
-using UnityEngine.XR.WSA.Input;
 
 namespace HoloToolkit.Unity.InputModule
```

Comment:
(Comment on the folder change) Interactions seems like it could be a base-level Input folder instead of inside Utilities. They're a pretty important part of input.

Reply:
I think scripts like this should really belong in the Examples or separate from the input system.

Don't get me wrong, they're important, but not part of the "Core" system.  More of an implementation of it.

Reply:
I agree with them being separate from the input system (like has been mentioned in #1032), but I disagree with them going into Examples. Yeah, it's not quite the "core" system, but it's still a collection of generic scripts that easily add functionality to an app.

Reply:
That's why I moved it into Utilites

Reply:
I'm down for moving this again when we do the reorg for MRDL